oxygen-addon.xml

```diff
@@ -0,0 +1,73 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xt:extensions xmlns:xi="http://www.w3.org/2001/XInclude"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:xt="http://www.oxygenxml.com/ns/extension"
+	xsi:schemaLocation="http://www.oxygenxml.com/ns/extension http://www.oxygenxml.com/ns/extension/extensions.xsd">
+
+	<!-- This @id value assumes that https://github.com/xspec/ community owns "xspec.io". -->
+	<xt:extension id="io.xspec.xspec">
+
+		<!-- To publish a new version, update this @href to a specific commit archive and increment
```

Comment:
The comments sounds like you change `@href` and `xt:version` in place, but I assumed you copy the whole `xt:extension` element and change `@href` and `xt:version` in the copy. If my assumption is correct, maybe it would be clearer to have the comment after the `xt:extensions` start tag and say something like, "To publish a new version, insert a new xt:extension element, set xt:location/@href to a specific commit archive, and increment xt:version."

Reply:
As described in https://github.com/xspec/xspec/pull/743#discussion_r352271007, I'm thinking to publish only the latest version (only one `<xt:extension>`) after v0.0.2. Not by technical reasons. Just to minimize our burden of development.

Reply:
Great.